### DISCUSSION TITLE
Continuous conversations

### DISCUSSION THREAD

**Discussion by alex-savin** [tetele/onju-voice-satellite#73] [Q&A]
Hey Folks,

What do you think about "Continuous conversations"?
I setup the voice assistant with "Extended OpenAI" and it has an option to ask user to provide some more details based on the original request. But all of the times I have to pronounce a magic word to continue conversation.
Could you please help me and point out how I can setup "Continuous conversations" option on the onju?

Thank you in advance,
Alex

**Comment by tetele** [comment]
The option you're mentioning has nothing to do with the Onju Voice, but with your HA pipeline setup. If you can use Extended OpenAI to achieve your goal, then the ESPHome integration in HA should automatically make it work with the Onju.

Are you experiencing different behavior? Tbf, I haven't tried conversation agents which are using continuous conversations so far.

**Comment by witold-gren** [comment]
It is ready 😀 please check this place: https://github.com/tetele/onju-voice-satellite/discussions/90

**Reply by alex-savin** [reply]
Thank you!
